java8-okhttp-gson/docs/ConfigurationApi.md

```diff
@@ -6,6 +6,7 @@ Method | HTTP request | Description
 ------------- | ------------- | -------------
 [**createConfiguration**](ConfigurationApi.md#createConfiguration) | **POST** /config | Create a new configuration
 [**getConfiguration**](ConfigurationApi.md#getConfiguration) | **GET** /config/{configName} | Get configuration
+[**updateConfiguration**](ConfigurationApi.md#updateConfiguration) | **POST** /config/{configName} | Partially update a configuration
```

Comment:
Is any of this documentation auto-generated when replacing the .swagger file from the updated blockchain-proof-ms and building?